pythonz/installer/pythoninstaller.py

```diff
@@ -63,7 +63,14 @@ def __init__(self, version, options):
                 logger.warning("Unsupported Python version: `%s`, trying with the following URL anyway: %s" % (version, self.get_version_url(version)))
             self.download_url = self.get_version_url(version)
         self.pkg = Package(version, options.type)
-        self.install_dir = os.path.join(PATH_PYTHONS, self.pkg.name)
+        if options.external_path:
+            if not os.path.isabs(options.external_path):
```

Comment:
Why this limitation? If a user runs `pythonz install --external my_pythons 3.5.0` in his home dir, I'd expect it to be installed, not to get an error.